### PR TITLE
Loadrunner integration

### PR DESCRIPTION
**Pull request by mihneavelin**
### What does this PR do?

Micro Focus LoadRunner Professional tile only listing.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by hestonhoffman** [comment]
Hi! I've opened up an [editorial review ticket](https://datadoghq.atlassian.net/browse/DOCS-4387) for the documentation team.

File: loadrunner/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+- Select the relevant metrics and add them to the **Selected metrics** list.
+
+- If you are defining metrics for a host group, define the aggregation method. Datadog applies the selected method to the metrics from all hosts.
+
+
+Click **OK**.
+
+
+The measurements are displayed in the **Resource Measurements** pane of the Datadog dialog box.
```
Comment (buraizu) [inline loadrunner/README.md:63]:
```suggestion
   ![snapshot][4]

   - Select the relevant metrics and add them to the **Selected metrics** list.

   - If you are defining metrics for a host group, define the aggregation method. Datadog applies the selected method to the metrics from all hosts.

   Click **OK**.

   The measurements are displayed in the **Resource Measurements** pane of the Datadog dialog box.

```

File: loadrunner/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+9. When done, click **OK** in the Datadog dialog box.
+
+10. After some brief processing, the Controller displays metrics in the Datadog graph.
+
+
+For metrics that are collected from multiple hosts (as indicated in the **Machine** column), each data point in the graph gives the aggregated value for all hosts. The aggregation method is indicated in the metric name, for example, **avg:metric1**.
```
Comment (buraizu) [inline loadrunner/README.md:70]:
```suggestion
For metrics that are collected from multiple hosts (as indicated in the **Machine** column), each data point in the graph gives the aggregated value for all hosts. The aggregation method is indicated in the metric name, for example, `avg:metric1`.

```

File: loadrunner/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Datadog graph
+
+The Analysis Datadog graph displays metrics fetched by the Datadog monitor. For details, see [Datadog monitor][5].
+
+When a metric is showing aggregated results from more than one host, the names of all hosts are included in the **Measurement** column in the legend. The aggregation method is indicated in the metric name, for example, **avg:metric1**.
```
Comment (buraizu) [inline loadrunner/README.md:76]:
```suggestion
When a metric is showing aggregated results from more than one host, the names of all hosts are included in the **Measurement** column in the legend. The aggregation method is indicated in the metric name, for example, `avg:metric1`.

```

File: loadrunner/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+4. Define the following to set up the connection to the Datadog server:
+
+![snapshot][2]
+
+-  **Site**. Select the Datadog site where you have your user account.
+
+-  **API key and application key**. These are generated in Datadog for your user account.
+
+Click **Connect**.
```
Comment (buraizu) [inline loadrunner/README.md:37]:
```suggestion
   ![snapshot][2]

   -  **Site**. Select the Datadog site where you have your user account.

   -  **API key and application key**. These are generated in Datadog for your user account.

   Click **Connect**.

```

File: loadrunner/README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+## Setup
+
+Set up the LoadRunner Professional Datadog monitor to view data from Datadog. For each Datadog host, you define the metrics for which you want to view data.
+
+
+You can also create a **host group** and define metrics for the group, as well as the aggregation function to use. Datadog applies the function to the results from the relevant hosts, and returns the response for each metric.
+
+
+### To set up the Datadog monitor:
```
Comment (buraizu) [inline loadrunner/README.md:19]:
```suggestion
### LoadRunner Professional Datadog monitor

```

File: loadrunner/README.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+1. In Controller, on the **Run** tab, select **Graphs > Datadog Graphs > Datadog**.
+
+The **Datadog graph** display box opens in the display area.
+
+2. In the graph display box, click **Add measurements**.
+
+3. In the **Datadog dialog box > Monitored Server Machines** pane, click **Add**. The **Connect to Datadog** dialog box opens.
+
+4. Define the following to set up the connection to the Datadog server:
```
Comment (buraizu) [inline loadrunner/README.md:29]:
```suggestion
4. Define the following to set up the connection to Datadog:

```

File: loadrunner/README.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+The measurements are displayed in the **Resource Measurements** pane of the Datadog dialog box.
+
+9. When done, click **OK** in the Datadog dialog box.
+
+10. After some brief processing, the Controller displays metrics in the Datadog graph.
+
+
+For metrics that are collected from multiple hosts (as indicated in the **Machine** column), each data point in the graph gives the aggregated value for all hosts. The aggregation method is indicated in the metric name, for example, **avg:metric1**.
+
+## Datadog graph
```
Comment (buraizu) [inline loadrunner/README.md:72]:
```suggestion
## Analysis Datadog graph

```

File: loadrunner/README.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+Click **Connect**.
+
+5. The **Select host** dialog box opens, listing the available hosts for the account.
+
+Select the host or hosts from which you want to pull metrics, and click **OK**.
+
+![snapshot][3]
+
+**Note:** If you select more than one host, this creates a host group for the metrics.
```
Comment (buraizu) [inline loadrunner/README.md:45]:
```suggestion
   Select the host or hosts from which you want to pull metrics, and click **OK**.

   ![snapshot][3]

   **Note:** If you select more than one host, this creates a host group for the metrics.

```

File: loadrunner/README.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+Set up the LoadRunner Professional Datadog monitor to view data from Datadog. For each Datadog host, you define the metrics for which you want to view data.
+
+
+You can also create a **host group** and define metrics for the group, as well as the aggregation function to use. Datadog applies the function to the results from the relevant hosts, and returns the response for each metric.
+
+
+### To set up the Datadog monitor:
+
+1. In Controller, on the **Run** tab, select **Graphs > Datadog Graphs > Datadog**.
```
Comment (buraizu) [inline loadrunner/README.md:21]:
```suggestion
1. In the LoadRunner Controller, on the **Run** tab, select **Graphs > Datadog Graphs > Datadog**.

```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for implementing those changes. I set this up in a local build and noticed a few additional styling/formatting issues. For example, sub-steps within a step were not indented. 

These suggestions should correct those issues, and also further clarify some sections which could cause confusion for users due to overlap with Datadog features (for example, Datadog also offers monitors, so I updated the title of the `To setup the Datadog monitor` section to clarify).

Thank you for your patience, and please let me know if you have any questions.

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Hi team! Excited for the integration. As discussed in our call today, the integration will also need to send data into Datadog. I am adding a hold label on the PR until the integration is updated to send data to Datadog. Please let me know if you have any questions!

**Pull request closed without merging**